<launch>
  <node cwd="node" name="lli_node" pkg="aauship_control" type="lli-node.py" output="screen"/>
  
  <node respawn="true" pkg="joy" type="joy_node" name="joy" output="screen">
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.000001" />
    <param name="coalesce_interval" value="0.1" />
  </node>
 
 <node cwd="node" name="joy_teleop_node" pkg="aauship_control" type="joy_teleop_node" output="screen"/> 
</launch>
